client/push/push_manager.js

```diff
@@ -30,12 +31,38 @@ export default class Push {
    */
   registerSubscription(subscription) {
     this._subscription = subscription;
+    const SERVER_URL = Util.getServerURL();
+    const queryUrl = `${SERVER_URL}/api/push/token/`;
+    let pushHeaders = new Headers({
+      'Content-Type': 'application/json'
+    });
+    let pushData = JSON.stringify({
+      // FIXME(daehyun): this userId should be replaced by real user id
+      userId: '1234',
```

Comment:
Let's use subscription.endpoint for userId temporary.
if we use fixed userId, it will always overwrite userId-subscription that already registered to server